src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
+
+  for (const key in robot) {
+    if (!obj[robot[key]]) {
```

Comment:
```suggestion
    if (!obj[robot[key]]) {
```
you can save this expression (obj[robot[key]]) in variable to write less code